lib/imagemagick.js

```diff
@@ -15,7 +15,8 @@
 var sprintf = require('util').format,
   log = require('sslog'),
   lru = require("lru-cache"),
-  child = require('child_process');
+  child = require('child_process')
+  playButtonPath = __dirname + '/../public/play-button.svg';
```

Comment:
Would it make sense to pull this out to some config file with asset definitions? Not sure if the code is structured in a way that makes it easy though.